ocs_ci/deployment/ibm.py

```diff
@@ -0,0 +1,548 @@
+
+        if len(sclassindevicesets) == 0:
+            return
+
+        # compare scnoprovisioner and sclassindevicesets
+        tmplist = []
+        for sclist in sclassindevicesets:
+            for sc in sclist:
+                if sc in scnoprovisioner:
+                    if sc not in tmplist:
+                        tmplist.append(sc)
+
+        scandlvlist = []
+        for sc in tmplist:
+            storage_class = ocp.OCP(
+                kind='StorageClass',
+                resource_name=sc
+            )
+            localvolumemeta = storage_class.get().get('metadata')
+            localvolume = localvolumemeta['labels']['local.storage.openshift.io/owner-name']
+            scandlvlist.append((sc[0], localvolume))
+
+        alldevicepaths = []
+        for lv in scandlvlist:
+            # Get localvolume and corresponding devices
+            localvolume = ocp.OCP(kind='LocalVolume',
+                                  namespace='local-storage',
+                                  resource_name=lv[1]).get().get('spec')
+            scdevices = localvolume['storageClassDevices']
+            devicepaths = [sub['devicePaths'] for sub in scdevices]
+            alldevicepaths.append(devicepaths[0][0])
+
+        # STEP 5
+        # delete storage cluster - all of them.
+        scdelete = 'oc delete -n openshift-storage storagecluster --all --wait=true'
+        # run_cmd(scdelete)
+
+        # STEP 6
+        spdelete = 'oc delete project openshift-storage --wait=true --timeout=5m'
+        logger.warning(spdelete)
+        # run_cmd(spdelete)
+
+        # STEP 7 & 9
+        nodes = ocp.OCP(kind='node').get().get('items', [])
+        worker_nodes = [
+            node for node in nodes if "cluster.ocs.openshift.io/openshift-storage"
+            in node['metadata']['labels']
+        ]
+        worker_node_names = [node['metadata']['name'] for node in worker_nodes]
+        logger.warning(worker_node_names)
+
+        # for worker_node in worker_node_names:
+        #     rookstring = f'oc debug node/'+ worker_node +
+        #                   ' -- chroot /host rm -rfv /var/lib/rook'
+        #     out = run_cmd(rookstring)
+        #     logger.warning(out)
+
+        #     for devicepath in alldevicepaths:
+        #          sgdiskstring = f'oc debug node/' + worker_node +
+        #                          ' -- chroot /host sgdisk --zap-all ' + deviepath
+        #          out = run_cmd(sgdiskstring)
+        #          logger.warning(out)
+
+        # STEP 8
+        for sc in scandlvlist:
+            lvdelete = 'oc delete localvolume -n local-storage --wait=true ' + sc[0]
+            logger.warning(lvdelete)
+            # run_cmd(lvdelete)
+            pvdelete = 'oc delete pv -l \
+                storage.openshift.com/local-volume-owner-name=' + sc[0] \
+                + ' --wait --timeout=5m'
+            logger.warning(pvdelete)
+            # run_cmd(pvdelete)
```

Comment:
most of the run_cmd are commented out, are you not deleting them?

Reply:
This is completed. The commented out code was because this was under testing.